src/sounds/soundsMap.test.ts

```
@@ -0,0 +1,114 @@
+import { describe, expect, it, vi, beforeEach } from 'vitest'
+import fs from 'fs'
```

ESLint

[error] `fs` import should occur before import of `vitest`

(import/order)